R/open.R

```diff
@@ -0,0 +1,36 @@
+
+#' Open main model definition file in text editor
+#'
+#' This is a light wrapper around [get_model_path()] to enable quickly jumping
+#' to a model definition file in RStudio or any other editor supported by
+#' [utils::file.edit()].
+#'
+#' @param .mod Model object or a file path to a model.
+#' @param ... Other arguments passed on to methods.
+#' @export
+open_model_file <- function(.mod, ...) {
```

Comment:
Hmm, actually, I don't see much reason to go with the method here.  (I'm not spotting where that was every explicitly proposed, so I'm not sure why I went that way.). `get_model_path` should return what's necessary for any model type and having `.character` method is a little silly because you'd just call `file.edit` if you already had the path.

I'll plan to rework this.